### PR TITLE
op-guide: make docker-compose doc runnable

### PR DESCRIPTION
**Pull request by siddontang**
Signed-off-by: siddontang <siddontang@gmail.com>

## What have you changed? (mandatory)

The origin docker-compose doc does nothing. Here add two examples and tell users how to run their own applications.

## What are the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

PTAL @lilin90

### CODE REVIEW

**Comment by ice1000** [comment]
Should we disable DCO for "verified" commits?

**Comment by queenypingcap** [comment]
I have verified the basic steps can work. @lilin90 please make this steps more beginner friendly later. Thanks!

File: docs/op-guide/deploy-tikv-using-docker-compose.md [hunk 1]
```diff
@@ -20,63 +20,143 @@ Make sure you have installed the following items on your machine:
...
 
-    ```bash
-    git clone https://github.com/pingcap/tidb-docker-compose.git
+```bash
+git clone https://github.com/pingcap/tidb-docker-compose.git
+```
+
+## Prepare cluster
+
+Here we will run a simple cluster with only 1 PD and 1 TiKV. The base docker compose config is:
```
Comment (queenypingcap) [inline docs/op-guide/deploy-tikv-using-docker-compose.md]: In this example, let's run a simple cluster with only 1 PD server and 1 TiKV server. See the following for the basic docker compose configuration file:

**Comment by lilin90** [comment]
@queenypingcap Got it. I'll update them soon.

File: docs/op-guide/deploy-tikv-using-docker-compose.md [hunk 2]
```diff
@@ -20,63 +20,143 @@ Make sure you have installed the following items on your machine:
...
+    depends_on:
+      - "pd0"
+    restart: on-failure
+```
+
+All the following example docker compose config file will contain this base config.
+
+## Example
+
+### [YCSB](https://github.com/pingcap/go-ycsb)
```
Comment (queenypingcap) [inline docs/op-guide/deploy-tikv-using-docker-compose.md]: -> Run [YCSB](https://github.com/pingcap/go-ycsb) to connect to the TiKV cluster:

File: docs/op-guide/deploy-tikv-using-docker-compose.md [hunk 3]
```diff
@@ -20,63 +20,143 @@ Make sure you have installed the following items on your machine:
...
+    Read empty for a
+    » insert a field0=0
+    Insert a ok
+    » read a
+    Read a ok
+    field0="0"
     ```
 
-6. Create and start the cluster using the `generated-docker-compose.yml` file.
+### [Titan](https://github.com/meitu/titan)
```
Comment (queenypingcap) [inline docs/op-guide/deploy-tikv-using-docker-compose.md]: -> Use [Titan](https://github.com/meitu/titan) to connect to the TiKV cluster via the Redis protocal:

File: docs/op-guide/deploy-tikv-using-docker-compose.md (hunk 2 above)
Comment (siddontang) [reply docs/op-guide/deploy-tikv-using-docker-compose.md]: is it too long for a title?

Comment (queenypingcap) [reply docs/op-guide/deploy-tikv-using-docker-compose.md]: Nope. Complete titles are good for search engines.

**Pull request merged**